packages/client/src/app/pages/portfolio/portfolio.component.html

```diff
@@ -4,6 +4,29 @@
             <img [src]="portUser.profile.avatar">
             <div class="port-username">
                 <h3>{{ portUser.username }}</h3>
+                <ng-container [ngSwitch]="determineProminentRole(portUser.roles)">
+                    <div class="role admin" *ngSwitchCase="'Admin'">
+                        Admin
+                    </div>
+                    <div class="role moderator" *ngSwitchCase="'Moderator'">
+                        Moderator
+                    </div>
+                    <div class="role chat-moderator" *ngSwitchCase="'ChatModerator'">
+                        Chat Moderator
+                    </div>
+                    <div class="role contributor" *ngSwitchCase="'Contributor'">
+                        Contributor
+                    </div>
+                    <div class="role work-approver" *ngSwitchCase="'WorkApprover'">
+                        Work Approver
+                    </div>
+                    <div class="role vip" *ngSwitchCase="'VIP'">
+                        VIP
+                    </div>
+                    <div class="role supporter" *ngSwitchCase="'Supporter'">
+                        Supporter
+                    </div>
+                </ng-container>
```

Comment:
This feels like a strong contender for its own component. Probably not something for this PR, but if it's an easy fix, I'd say throw it in

Reply:
a strong contender, yes! i’ll put a note to do that for 0.5